### PR TITLE
Fixes #122 - Upgraded hyrax to the latest commit on master branch.

### LINKED ISSUE #122: Upgrade Hyrax to latest commit
### Descriptive summary

- [ ] Please update to the most available commit for Hyrax.
- [ ] Please provide a summary of the changes in the PR so the entire team understands the context of the changes.

### Rationale

To stay up to date with the latest Hyrax developments upstream.

### References

- #111 ( @lsitu - notes a File ingest job change will potentially impact )

### PR DESCRIPTION
**Pull request by lsitu**
Fixes #122 ; refs #122

Upgraded hyrax to the latest commit on master branch.

Changes proposed in this pull request:
* Upgrade it to the latest commit on hyrax master branch, , as of commit https://github.com/samvera/hyrax/commit/9f117a32a9fdd42a7884c3c5ee59997ad1e8f2d4 at the time beings.
* Exclude the hyrax “access” key from nested attributes processing to avoid the error "undefined method `each_with_index' for "edit":String."
* Fixe the collection form loading error "NoMethodError: undefined method `apply_depositor_metadata' for nil:NilClass".
* Added the new table job_io_wrappers.

@ucsdlib/developers - please review

### CODE REVIEW

File: app/controllers/hyrax/collections_controller.rb [hunk 1]
```diff
@@ -2,6 +2,8 @@ module Hyrax
   class CollectionsController < ApplicationController
     include Hyrax::CollectionsControllerBehavior
     include Hyrax::BreadcrumbsForCollections
+    layout :decide_layout
+    load_and_authorize_resource except: [:index, :show, :create], instance_name: :collection
```
Comment (mcritchlow) [inline app/controllers/hyrax/collections_controller.rb:6]: why is `:create` excluding from `load_and_authorize_resource`?

**Review by mcritchlow — COMMENTED** [review]
Just one question on my end. Otherwise, LGTM. Thanks @lsitu 👍

File: app/controllers/hyrax/collections_controller.rb (hunk 1 above)
Comment (lsitu) [reply app/controllers/hyrax/collections_controller.rb:6]: @mcritchlow: Good question! I think action `:create` is expected to have its's own authorization before creating a new collection,  see [collections_controller_behavior.rb#L103](https://github.com/samvera/hyrax/blob/master/app/controllers/concerns/hyrax/collections_controller_behavior.rb#L103).

**Comment by mcritchlow** [comment]
Ah, that makes much more sense. Thanks for the pointer!

**Comment by VivianChu** [comment]
👍

**Review by VivianChu — APPROVED** [review]

**Pull request merged**